### PR TITLE
New soldr events

### PR DESCRIPTION
**Pull request by TorrentKatten**
There are some cleanup needed, but pushes events with the metedata that we need.

### CODE REVIEW

File: lib/rabbit_client/bunny_client.rb [hunk 1]
```diff
@@ -10,6 +10,34 @@ def self.publish_messages(url, exchange_name, exchange_type, messages)
       end
     end
 
+    def self.publish_message(url, exchange_name, exchange_type, message, routing_key, batch, origin)
```
Comment (knyl) [inline lib/rabbit_client/bunny_client.rb]: Could we use keyword arguments in this method?

File: lib/rabbit_client/consumer.rb [hunk 2]
```diff
@@ -13,10 +13,10 @@ class << self
       def configure(opts)
         @queue_name = opts[:queue]
         configuration = {
-          exchange: opts[:exchange],
-          exchange_type: opts[:exchange_type] || 'fanout',
-          heartbeat: opts[:heartbeat] || 60,
-          arguments: { :'x-dead-letter-exchange' => "#{@queue_name}-retry" }
+        exchange: opts[:exchange],
```
Comment (knyl) [inline lib/rabbit_client/consumer.rb]: Was this a rubocop warning? Anyways, I think we should try to make rubocop/rubymine indent this as it was previously.

File: lib/rabbit_client/publisher.rb [hunk 3]
```diff
@@ -14,6 +14,18 @@ def self.publish_messages(url, exchange_name, messages, opts = {})
       end
     end
 
+    def self.publish_message(url, exchange_name, exchange_type,routing_key, batch, origin, message, opts = {})
```
Comment (knyl) [inline lib/rabbit_client/publisher.rb]: Keyword arguments would be nice here as well.

File: spec/rabbit_client/publisher_spec.rb [hunk 4]
```diff
@@ -3,27 +3,70 @@
...
+      expect(RabbitClient::BunnyClient)
+      .to(receive(:publish_messages))
+      .with url, exchange, exchange_type, messages
+      RabbitClient::Publisher.publish_messages url,
+                                               exchange,
+                                               messages,
+                                               exchange_type: exchange_type
+    end
+
+    it 'publishes messages to exchange to topic' do
```
Comment (knyl) [inline spec/rabbit_client/publisher_spec.rb]: There is two of this testcase. The more the merrier? :D

File: lib/rabbit_client/bunny_client.rb (hunk 1 above)
Comment (TorrentKatten) [reply lib/rabbit_client/bunny_client.rb]: Sounds like a good idea, didnt think of that.

File: lib/rabbit_client/bunny_client.rb [hunk 5]
```diff
@@ -10,6 +10,34 @@ def self.publish_messages(url, exchange_name, exchange_type, messages)
...
+      connect url, exchange_name, exchange_type do |exchange|
+
+        current_time = Time.now.getutc
+        organization = message.organization
+
+        message_headers = {}
+
+        message_headers[:routing_key] = routing_key
+        message_headers[:organization] = organization
+        message_headers[:batch] = batch
```
Comment (knyl) [inline lib/rabbit_client/bunny_client.rb]: We need to discuss what we mean with 'batch', and what implications it should have

**Pull request merged**